### PR TITLE
Event duplication wrong time (EXPOSUREAPP-6439)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Fixing wrong date/time displayed in `QrDetailFragment`, `TraceLocationsFragment`, `TraceLocationCreateFragment` 
- Add instrumentation tests

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -162,6 +162,7 @@ commands:
             -no-boot-anim
             -memory 2048
             -nojni
+            -timezone Europe/Berlin
```
Comment (mtwalli) [inline .circleci/config.yml:165]: why , I recall some situations where the different timezone in CI helped us to spot some issues and you already doing that in the tests that require this time zone

**Review by mtwalli — COMMENTED** [review]
LGTM, tested and I confirm the fix

File: .circleci/config.yml (hunk 1 above)
Comment (jurajkusnier) [reply .circleci/config.yml:165]: I was testing multiple approaches, I guess this is not necessary anymore, I'll try to remove it and check if the test pass

**Review by mtwalli — APPROVED** [review]

**Review by BMItr — APPROVED** [review]
lgtm, tested on pixel 2 device

**Pull request merged**